common/src/main/java/com/wynntils/overlays/minimap/MinimapOverlay.java

```diff
@@ -93,6 +98,21 @@ public MinimapOverlay() {
                 new OverlaySize(DEFAULT_SIZE, DEFAULT_SIZE));
     }
 
+    @SubscribeEvent
+    // Using TickEvent for smooth zooming
+    public void onTick(TickEvent event) {
+        long window = McUtils.window().getWindow();
+        boolean zoomOut = InputConstants.isKeyDown(window, InputConstants.KEY_EQUALS)
+                || InputConstants.isKeyDown(window, InputConstants.KEY_ADD);
+        boolean zoomIn = InputConstants.isKeyDown(window, InputConstants.KEY_MINUS)
+                || InputConstants.isKeyDown(window, GLFW.GLFW_KEY_KP_SUBTRACT);
```

Comment:
This is not a screen; you can add keybinds. Do so

Reply:
which keys should be the default ones? or should I add 4 keybinds?

Reply:
the normal ones, not the numpad version.

Reply:
can I register them in the overlay?

Reply:
in the feature only